### DISCUSSION TITLE
How to get U.S. Unemployment Rate in investing.com to use investpy ?

### DISCUSSION THREAD

**Discussion by zkdlfrlwl2** [alvarobartt/investpy#594] [Q&A]
Hi, 

I want to get U.S. Unemployment Rate in investing.com to use investpy.

But, I can't find that way. 

Anyone know about ? 

Thanks.

**Comment by jarvisNim** [comment]
Umm...

df = inv.economic_calendar(countries=['United States'], from_date='05/08/2022', to_date='06/08/2022', importances=['high'])
df.loc[df['event'].str.contains('Unemployment', case=False)]

You can find it in Calendar Event, But It's hard to search directly. so I used to save into sqlite3 to search these things.
sorry for poor english.

**Reply by zkdlfrlwl2** [reply]
I solved this problem to use selenium.

Thanks for your reply!
